pipeline/src/node_properties/snapshot_properties.rs

```diff
@@ -3,13 +3,35 @@ use serde_with::skip_serializing_none;
 use std::path::PathBuf;
 use url::Url;
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "location", rename_all = "snake_case")]
+pub enum SnapshotProperties {
+    Local(LocalSnapshotProperties),
+    LumeoCloud(LumeoCloudSnapshotProperties),
+}
+
 #[skip_serializing_none]
-#[derive(Debug, Default, Clone, PartialEq, Serialize, Deserialize)]
-pub struct SnapshotProperties {
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct CommonSnapshotProperties {
+    pub min_duration: Option<u64>,
+    pub max_duration: Option<u64>,
+    pub max_size: Option<u64>,
+    pub retention_duration: Option<u64>,
+    pub webhook_url: Option<Url>,
+    pub trigger: Option<String>,
+}
```

Comment:
min_duration, max_duration and max_size not applicable to snapshots.

Reply:
Oops, only checked that I don't have too few fields in there, didn't check for extraneous ones from ClipProperties.